### PR TITLE
Fix app resizing

### PR DESCRIPTION
**Pull request by kunkuma**
This PR includes fixing the app resizing issue

### CODE REVIEW

File: src-tauri/tauri.conf.json [hunk 1]
```diff
@@ -1,46 +1,46 @@
...
+                "icons/icon.ico"
+            ]
+        },
+        "security": {
+            "csp": null
+        },
+        "windows": [
+            {
+                "fullscreen": false,
+                "resizable": false,
```
Comment (mdvmeijer) [inline src-tauri/tauri.conf.json:39]: Does this mean that you cannot resize the app window at all?

**Review by mdvmeijer — COMMENTED** [review]
How exactly has the behaviour on Windows changed with this fix? It includes some Windows specific things, so I cannot test it accurately

File: src-tauri/tauri.conf.json (hunk 1 above)
Comment (kunkuma) [reply src-tauri/tauri.conf.json:39]: Yes, you can't change it at all. But that was for testing purposes. I'll revert.

**Comment by kunkuma** [comment]
> How exactly has the behaviour on Windows changed with this fix? It includes some Windows specific things, so I cannot test it accurately

Yes. Those windows specific things actually made everything fit. I misread "windows" as "window" since tauri has a window configuration to allow manipulating the window. https://tauri.app/v1/api/js/window.

**Comment by kunkuma** [comment]
@ketelaar @mdvmeijer I just pushed a new fix. This should work well. Please let me know what you think.

**Comment by ketelaar** [comment]
> @ketelaar @mdvmeijer I just pushed a new fix. This should work well. Please let me know what you think.

Still the same, I get a horizontal scrollbar as well now

image

**Comment by kunkuma** [comment]
> > @ketelaar @mdvmeijer I just pushed a new fix. This should work well. Please let me know what you think.
> 
> Still the same, I get a horizontal scrollbar as well now
> 
> image

afbeelding

That's strange. This is what I get. I'll take another look.

File: .gitignore [hunk 2]
```diff
@@ -24,3 +24,4 @@ dist-ssr
 *.sw?
 # environment variables
 .env
+*.toml
```
Comment (ketelaar) [inline .gitignore:27]:
```suggestion
```
This should not be added in the .gitignore, since Cargo.toml is used for dependency management

**Comment by ketelaar** [comment]
The body and app divs are 800x576. Note that I can't directly get this value since the inspector cannot be detached from the window for WebKit, but I moved the inspector around without resizing the window.

**Comment by kunkuma** [comment]
@ketelaar @mdvmeijer I just updated with the changes we discussed.

**Review by ketelaar — APPROVED** [review]
Looks good

**Pull request merged**